GladiusEx.lua

```diff
@@ -748,7 +748,7 @@ function GladiusEx:CHAT_MSG_SYSTEM(event, msg)
     if InCombatLockdown() then
         local name = string.gsub(msg, " has left the battle", "")
         for frame in pairs(self.buttons) do
-            if frame and string.find(UnitName(frame), name) and not UnitExists(frame) then
+            if frame and UnitName(frame) and string.find(UnitName(frame), name) and not UnitExists(frame) then
```

Comment:
Why are we doing UnitExists on a frame? Is frame a unit?

Reply:
for frame in pairs(self.buttons) means the key value regardless whether the key is a table or not
if it was
for frame, setting in pairs(self.buttons) then frame would be table not just the 'value' of the key